src/Service/ResourceService.php

```diff
@@ -49,7 +50,7 @@ public function getWhitelistedResources($permission, $whitelistKey): array
     {
         $this->metricsHelper->incMethodTotal(__METHOD__, MetricsHelper::INVOKE);
 
-        $info = $this->aakResourceRepository->getOnlyWhitelisted($permission, $whitelistKey);
+        $info = $this->aakResourceRepository->getOnlyWhitelisted($permission, $whitelistKey, $this->excludesResources);
```

Comment:
Typo if you wish to fix
```suggestion
        $info = $this->aakResourceRepository->getOnlyWhitelisted($permission, $whitelistKey, $this->excludedResources);
```